src/api/libcellml/types.h

```diff
@@ -16,20 +16,24 @@ limitations under the License.
 
 #pragma once
 
+#include <any>
+#include <map>
 #include <memory>
 
+#include "libcellml/enums.h"
+
 namespace libcellml {
 
 // General classes.
 class Annotator; /**< Forward declaration of Annotator class. */
-using AnnotatorPtr = std::shared_ptr<Annotator>; /**< Type definition for shared annotator pointer. */
+using AnnotatorPtr = std::shared_ptr<Annotator>; /**< Type definition for @c std::shared Annotator pointer some othe text. */
 class Generator; /**< Forward declaration of Generator class. */
```

Comment:
typo "othe" ... and also ... huh?

Reply:
Playing around with Doxygen documentation.